package/MDAnalysis/analysis/legacy/x3dna.py

```diff
@@ -162,7 +162,10 @@ def mean_std_from_x3dnaPickle(profile):
         stretch, stagger, buckle, propeller, opening, shift, slide, rise, tilt,
         roll, twist]``.
 
+    .. deprecated:: 2.7.0
+        X3DNA will be removed in 3.0.0.
     """
+    warnings.warn("X3DNA module is deprecated and will be removed in MDAnalysis 3.0, see #3788", category=DeprecationWarning)
```

Comment:
Thanks for testing this locally, that's probably good enough.

Reply:
we have a decorator for this - please see https://github.com/MDAnalysis/mdanalysis/blob/5fa0d27571f49516f4f68bd200a9b6c705abf99d/package/MDAnalysis/analysis/hole2/hole.py#L50-L52 for an example

Reply:
ah ta.

Reply:
RE: 1 if the code is not usable and we are removing it i don't really want to invest any energy in finding out why / how. Is it okay just to put the warnings in as is? Let me know wither way.

Reply:
@hmacdope - my point here is "what's the point in putting in a warning if it won't even work locally". If you can get the warnings locally then it's fine, otherwise I'd just add in the docstring changes and call it a day.

Reply:
Okay you can get the warnings on *before* whatever is broken, at least it may warn the (possibly 0) people that have a niche setup that this works for this that it will be removed.